spring-security-jwt/src/test/java/org/springframework/security/jwt/crypto/cipher/EllipticCurveKeyData.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2006-2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package org.springframework.security.jwt.crypto.cipher;
+
+import org.springframework.security.jwt.codec.Codecs;
+
+import java.math.BigInteger;
+
+/**
+ * @author Michael Duergner
+ */
+public class EllipticCurveKeyData {
```

Comment:
Please remove this class and put test data directly in `EllipticCurveVerificationTests`